connectors/katalog/example/asset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: katalog.m4d.ibm.com/v1alpha1
+kind: Asset
+metadata:
+  name: data-csv
+spec:
+  secretRef: 
+    name: data-csv-creds
+  details:
+    dataFormat: csv
+    connection:
+      type: s3
+      s3:
+        endpoint: "http://minio:9000"
+        bucket: demo
+        objectKey: data.csv
+  security:
+    geography: us-south
```

Comment:
These are data set level metadata? If so, perhaps we should call them something like datasetMetadata and componentMetadata?

Reply:
Will do